istio-telemetry/mixer-telemetry/templates/deployment.yaml

```diff
@@ -87,7 +87,7 @@ spec:
 {{- if .Values.global.logAsJson }}
           - --log_as_json
 {{- end }}
-{{- if .Values.global.useMCP }}
+{{- if .Values.mixer.telemetry.useMCP }}
```

Comment:
At least respect global - if global is false it typically means MCP is not installed. 

Any reason to have a telemetry-specific setting ?

Reply:
Pilot uses a local `useMCP` here: https://github.com/istio/installer/blob/master/istio-control/istio-discovery/templates/configmap.yaml#L119. I was trying to make the variable being used in Pilot and Mixer more consistent, but if the recommended way is to remove local `useMCP` and stick with the global one, I'm fine with that as well.

Reply:
I think it's better as you have it, globals are bad.
( and besides they are no longer really global - each component is now a separate module )

Reply:
ok, let me revert the changes so that we avoid using globals.